### PR TITLE
Clone Google Search Page

### LINKED ISSUE #1: Clone Google Page

### PR DESCRIPTION
**Pull request by ydaniju**
Clone Google Search Page

We want to have the google search page of our own. So,
we did anyway.

This feature was created by:
* creating folder for each group of files

Resolves #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
+
```
Comment (ydaniju) [inline .gitignore]: Wow! I like this!

File: readme.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Learning HTML
```
Comment (ydaniju) [inline readme.md]: Can we remove the Google Image from our codebase and use a CDN?

Comment (aoalaka) [inline readme.md]: Add more info to your **readme** file

Comment (aoalaka) [reply readme.md]: add more info to your **readme** file for usto read

**Comment by ydaniju** [comment]
Alright @aoalaka I'd create an issue for that.

**Pull request merged**